zeq/detail/port.h

```diff
@@ -19,7 +20,7 @@ namespace detail
 uint16_t getPort( const std::string& name );
 
 /** @return a random port number. */
-uint16_t getRandomPort();
+ZEQ_API uint16_t getRandomPort();
```

Comment:
It's used in all the tests, hence the export is needed there. The detail is not installed, so nobody outside of ZeroEQ can use it. It's more fishy that this function is used in the tests or even that ZeroEQ provides a random port generation function :)